src/Annotation/Annotation.scss

```diff
@@ -0,0 +1,72 @@
+@import "variables";
+
+@mixin triangle($color, $direction) {
```

Comment:
Is this the same triangle that's used by `Tooltip` and `Popover`? If not, would it be possible to use the same arrow as `Tooltip`/`Popover`? I believe the triangle for those components is coming from react-bootstrap.

Reply:
No, it's not the same, I though I'd try to implement new arrow after seeing that both `Popover` and `Tooltip` use arrow from react-bootstrap for couple of reasons:
1. Both arrows from react-bootsrap contain variables specific to the component they relate to (e.g. [Tooltip arrow](https://github.com/twbs/bootstrap/blob/main/scss/_tooltip.scss)), I can't override them since they are used in Paragon's  components (I believe I would need to use `@extend` and override them directly in class definition)
2. They both don't support color variants, so I would need to modify them

I thought that if I try to use bootstrap's arrow I would end up copying a lot of stuff to customize it for `Annotation` component design spec and decided that maybe it would be better to implement new one. I can switch to react-bootstrap if you prefer it more.